python-modules/cis_change_service/cis_change_service/api.py

```diff
@@ -0,0 +1,83 @@
+import logging
```

Comment:
careful with the scope, thats a lot of defines ;-)

Reply:
All the logging is put in haphazard (like this) at present because it'll be replaced with cis_logging at the end of the build.

Reply:
i meant more that style wise you often have a lot of "from x import z" and end up with a lot of "z", which generally makes it more likely to mistakenly override variables (specially in the future). style choice